requirements.txt

```diff
@@ -0,0 +1,2 @@
+datajoint==0.13.dev2
```

Comment:
Does it need to use 0.13.dev2? Would 0.12.7 work for this package?

Reply:
needs 0.13.dev2, as that includes a bugfix in `schema.save()`

Reply:
could potentially monkey-patch the bugfix, not sure if that's worth the effort given 0.13 is coming soon